WeightProducer/python/MCSampleValues.py

```diff
@@ -999,141 +817,164 @@ class MCSampleValuesHelper():
+                XS_13TeV=12.17, XSSource_13TeV="XSDB (unknown)",
             ),
         },
         "WWTo2L2Nu" : {
             "CrossSection" : XSValues(
-                XS_13TeV=12.178, XSSource_13TeV="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Diboson, WW>2l2v NNLO",
+                XS_13TeV=11.09, XSSource_13TeV="XSDB (NLO)",
             ),
         },
         "WGJets_MonoPhoton_PtG-40to130" : {
             "CrossSection" : XSValues(
-                XS_2016=12.68,  XSSource_2016="XSDB (LO)",
-                XS_2017=12.93, XSSource_2017="XSDB (LO)",
-                XS_2018=12.95, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=19.77,  XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
         "WGJets_MonoPhoton_PtG-130" : {
             "CrossSection" : XSValues(
-                XS_2016=0.6578, XSSource_2016="XSDB (LO)",
-                XS_2017=0.7158, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=0.7153, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=0.8092, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
+        #TODO
         "WWTo1L1Nu2Q" : {
             "CrossSection" : XSValues(
                 XS_2016=45.68, XSSource_2016="XSDB (LO)",
                 XS_2017=80.74, XSSource_2017="XSDB (LO)",
                 XS_2018=81.46, XSSource_2018="GenXSecAnalyzer",
             ),
         },
+        #TODO
         "WZTo1L1Nu2Q" : {
             "CrossSection" : XSValues(
                 XS_2016=10.73, XSSource_2016="XSDB (LO)",
                 XS_2017=11.66, XSSource_2017="XSDB (LO)",
                 XS_2018=11.76, XSSource_2018="GenXSecAnalyzer",
             ),
         },
+        #TODO
         "WZTo1L3Nu" : {
             "CrossSection" : XSValues(
                 XS_2016=3.054, XSSource_2016="XSDB (LO)",
                 XS_2017=3.294, XSSource_2017="GenXSecAnalyzer",
                 XS_2018=3.322, XSSource_2018="GenXSecAnalyzer",
             ),
         },
+        "WZTo3LNu" : {
+            "CrossSection" : XSValues(
+                XS_13TeV=5.213, XSSource_13TeV="XSDB (NLO)",
+            ),
+        },
+        "WZTo2Q2Nu" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=6.901, XSSource_13TeV="GenXSecAnalyzer",
+            ),
+        },
+        "WWTo4Q_4f" : {
+           "CrossSection" : XSValues(
+                 XS_13TeV=51.07, XSSource_13TeV="GenXSecAnalyzer",
+            ),
+        },
+        #TODO
         "ZGTo2NuG" : {
             "CrossSection" : XSValues(
                 XS_2016=28.04, XSSource_2016="XSDB (unknown)",
             ),
         },
         "ZZTo2L2Nu" : {
             "CrossSection" : XSValues(
-                XS_2016=0.5644, XSSource_2016="GenXSecAnalyzer",
-                XS_2017=0.6008, XSSource_2017="GenXSecAnalyzer",
-                XS_2018=0.6008, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=0.9738, XSSource_13TeV="XSDB (NLO)",
```

Comment:
Seems a lot higher than before. Is that simply by going from LO to NLO?

Reply:
See hypernews thread [1] for some insight on this ZZTo2L2Nu sample.

[1] https://hypernews.cern.ch/HyperNews/CMS/get/generators/5172.html